Anudle-sy/pest-smart
# Create Next.js App

Summary:
- this PR replaces the initial create-react-app with a Next.js app (similar to the create-react-app, this bootstraps the app for you + handles routing)
- Next.js is an open source frame work developed by a third party in conjunction with Facebook & Google. This is the framework that the official React documentation recommends using (over generic Node.js server side)
- in order to test, your IP address must be whitelisted in MongoDB Atlas instance. Contact Molly to get whitelisted

To Test:
- cd into the right directory (it might be nested depending on how you set up the repo)
- Run `npm install`
- Run `npm run dev`
- Navigate to http://localhost:3000/
- Verify that the page looks like this: 
![image](https://user-images.githubusercontent.com/81973596/116167434-d22f1080-a6c5-11eb-9705-a3b6842f9c90.png)
- Navigate to http://localhost:3000/movies
- Verify that the page loads the sample moves dataset from MongoDB
![image](https://user-images.githubusercontent.com/81973596/116167477-ea069480-a6c5-11eb-834e-ce971c435238.png)

Reference:
[Create Nextjs documentation](https://nextjs.org/learn/basics/create-nextjs-app)
